elements/lisk-p2p/src/p2p_types.ts

```diff
@@ -53,55 +63,29 @@ export interface P2PInternalState {
 	readonly productivity?: number;
 	readonly reputation?: number;
 	readonly connectionKind?: ConnectionKind;
-	readonly advertiseAddress: boolean;
 }
 
-export interface P2PPeerInfo {
+export interface P2PBasicPeerInfo {
 	// String to uniquely identify each peer
-	readonly peerId: string;
+	readonly id: string;
 	readonly ipAddress: string;
-	readonly wsPort: number;
-	readonly sharedState?: P2PSharedState;
-	readonly internalState?: P2PInternalState;
 }
 
-export interface P2PPeersCount {
-	readonly outboundCount: number;
-	readonly inboundCount: number;
-}
-
-// P2PPeerInfo and P2PNodeInfo are related.
-// P2PNodeInfo is the outbound info from our node.
-export interface P2PNodeInfo extends P2PSharedState {
-	readonly protocolVersion: string;
-	readonly os: string;
-	readonly nethash: string;
-	readonly wsPort: number;
-	readonly advertiseAddress: boolean;
-	readonly nonce: string;
-}
-
-// This is a representation of the inbound peer object according to the current protocol.
-// TODO later: Switch to LIP protocol format.
-// TODO: Include peerId as field
-export interface ProtocolPeerInfo {
-	// To support the existing protocol
-	readonly ipAddress: string;
-	readonly wsPort: number;
-	// tslint:disable-next-line: no-mixed-interface
-	readonly [key: string]: unknown;
+export interface P2PPeerInfo extends P2PBasicPeerInfo {
```

Comment:
This is like `P2PDisocveredPeerInfo`

Reply:
Can you elaborate a bit more? `P2PDiscoveredPeerInfo` doesn't exist anymore.

Reply:
I meant that we had this old types -> new types
`P2PPeerInfo` -> `P2PBasicPeerInfo`
`P2PDiscoveredInfo` -> `P2PPeerInfo`

Reply:
They weren't exactly the same. `P2PPeerInfo` contains now basic, shared and internal data.